app/controllers/users_controller.rb

```diff
@@ -142,8 +143,29 @@ def access_list_approvals
   def email
   end
 
+  # GET /facilities/:facility_id/users/:id/edit
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  # PUT /facilities/:facility_id/users/:id
+  def update
+    @user = User.find(params[:id])
+    if @user.update_attributes(edit_user_params)
```

Comment:
I can add it and see how many tests it breaks but I didn't even think you had to explicitly include it?